extensions/src/ACRE2Core/FilterOcclusion.cpp

```diff
@@ -6,12 +6,13 @@
 #include <cmath>
 #include <math.h>
 
-#define TS_SAMPLE_RATE 48000
-#define CUTTOFF_FREQ    15000
+#define TS_SAMPLE_RATE_Hz 48000
+#define CUTTOFF_FREQ_Hz    15000
```

Comment:
```suggestion
#define CUTTOFF_FREQ_Hz   15000
```

Reply:
Macros are usually identified by being all upper case. The z is not.

Reply:
This is accepted when trying to define units. mHz is not the same as MHz and so on.